README.md

```diff
@@ -235,11 +237,11 @@ A different qcow2 image (e.g., one with an `intel` kernel) can be used by settin
 
 ### 6.3 Secure Boot TD
 
-To run a TD VM with secure boot enabled, use the `tdvirsh` tool with
+To run a TD with secure boot enabled, use the `tdvirsh` tool with
 dedicated libvirt configuration:
 
 	cd tdx/guest-tools
-	XML_TEMPLATE=trust_domain-sb.xml.template ./tdvirsh new
+	./tdvirsh new -i trust_domain-sb.xml.template
```

Comment:
Is that rather `-t` instead of `-i`

Reply:
Yep, sorry for that. Fixed in new commit. 
Please squash all my commits on merge using the commit message only from the first commit.